grep-00XX-relicense-volk.md

```diff
@@ -0,0 +1,86 @@
+# GREP [00XX] -- [Relicense VOLK]
+
+- Original Author: [Johannes Demel <demel@uni-bremen.de>]
+- Champion: [Johannes Demel <demel@uni-bremen.de>]
+- Status: Draft
+
+History:
+- 18-Mar-2021: Initial Draft
+
+## Abstract
+
+We have several community requests to relicense VOLK under a more permissive license.
```

Comment:
I recommend framing this not from the perspective of "some people want a different license" (which is likely to still be the case after this GREP, actually), but rather from what the goal of taking this action is -- namely, removing barriers to adoption, growing the user & contributor base of VOLK, and spreading the value of Free & Open Source Software in the SIMD community, which so far is dominated by non-FOSS.

Reply:
Thanks! I adopted your comments.